cli/src/pcluster/templates/imagebuilder_stack.py

```diff
@@ -483,6 +483,40 @@ def _add_imagebuilder_components(self, build_tags, lambda_cleanup_policy_stateme
         if self.config.build.components:
             self._add_custom_components(components, lambda_cleanup_policy_statements, components_resources)
 
+        if not disable_pcluster_component:
```

Comment:
we need a setting to disable this new component

Reply:
done